server/middlewares/loginValidation.js

```diff
@@ -0,0 +1,16 @@
+const joi = require('joi');
+// login input Validation
+const loginValidation = (req, res, next) => {
+  const schema = joi.object({
+    email: joi.string().email().required(),
+    password: joi.string().alphanum().min(6).required(),
+  });
```

Comment:
create your schema in a separated file in the utils/validation folder  then import it here (Clean Code)